Modules/gcmodule.c

```diff
@@ -929,9 +929,8 @@ delete_garbage(struct _gc_runtime_state *state,
                 Py_INCREF(op);
                 (void) clear(op);
                 if (PyErr_Occurred()) {
-                    PySys_WriteStderr("Exception ignored in tp_clear of "
-                                      "%.50s\n", Py_TYPE(op)->tp_name);
-                    PyErr_WriteUnraisable(NULL);
+                    _PyErr_WriteUnraisableMsg("Exception ignored in tp_clear"
```

Comment:
I think that `"Exception ignored"` or `"Exception ignored in"` is the constant part, so you can pass only the variable part. See at `Py_EnterRecursiveCall()` as a pattern.

Reply:
Done with "Exception ignored"